source/common/image_processing.py

```diff
@@ -0,0 +1,155 @@
+import cv2
+import pandas as pd
+import os
+import common.image_properties as ip
+
+def getBpmFromColor(x_axis, y_axis, image):
```

Comment:
Please adhere to [our coding guidelines](https://github.com/Holandsoest/art-to-music/wiki/Guidelines-for-our-python-project#naming-conventions):  
- Use `snake_case` for filenames, directories, functions, methods, variables and fields

Reply:
I will fix this in the code